### DISCUSSION TITLE
Welcome to ockovani-covid Discussions!

### DISCUSSION THREAD

**Discussion by msusicky** [msusicky/ockovani-covid#540] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by karelpelan** [comment]
Dobrý den,
dostal jsem odkaz na váš web k očkování, kde je uváděna doba čekání na rezervaci.
Konkrétně u našeho očkovacího místa Edumed - očkovací centrum Jaroměř je uveden nesmysl, který vnímáme jako nás poškozující.
U čekání na rezervaci uvádíte 8 týdnů, přitom skutečnost je 1 den! To lze snadno ověřit na stánkách MZCR - očkovací místa a je to nakonec vidět i ve Vašem podrobnějším přehledu v záložce volná místa.

Žádám Vás tímto o okamžitou nápravu této nepravdivé informace.

Karel Pelán

**Reply by jan-stanek** [reply]
Dobrý den,
děkujeme za upozornění. Problém byl s velkým množstvím registrací, které byly vytvořeny už v březnu nebo dubnu, ale termín si rezervovaly až teď, výrazně tak navyšovaly průměr. Od zítřka tyto staré registrace nebudeme započítávat a průměrná doba čekání klesne.

Jan Staněk

**Comment by karelpelan** [comment]
Dobrý den,
oceňuji rychlost opravy. Zmíněné staré registrace budou zmatečné, protože očkovací centrum provozujeme až od poloviny dubna, takže v březnu se k nám nikdo nemohl registrovat ani teoreticky.

KP
